src/js/jsx/search/SearchBar.jsx

```diff
@@ -180,11 +180,18 @@ define(function (require, exports, module) {
 
                     var title = option.title.toLowerCase(),
                         category = option.category || [];
+ 
+                    if (option.id.indexOf("FILTER") === 0) {
```

Comment:
I changed filters to have type of "filter" instead of "item," which gets rid of this string comparison, but I don't think there's a simple workaround for `_handleChange` in `SearchBar.jsx`. We could look up the object by ID in the search options, but the string comparison seems more efficient.
https://github.com/adobe-photoshop/spaces-design/blob/master/src/js/jsx/search/SearchBar.jsx#L110